euth/actions/signals.py

```diff
@@ -1,18 +1,75 @@
+from django.apps import apps
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
 from euth.actions import emails
+from euth.modules.models import Item
+from euth.projects.models import Project
 
 from . import verbs
 from .models import Action
 
 
+def notify_creator(instance):
+    if hasattr(instance.target, 'creator'):
+        creator = instance.target.creator
+        if creator.get_notifications and not creator == instance.actor:
+            emails.notify_users_on_create_action(instance, [creator.email])
+
+
+def notify_moderators(instance):
+    if str(instance.target_content_type) == 'project':
```

Comment:
In my personal opinion `instance.target_content_type.model_class() is Project` would be preferable, but that is certainly only a question of style.